### PR TITLE
Add stories for search bar

### LINKED ISSUE #369: Add search bar components to Storybook
- [x] SearchBar.svelte
- [x] SearchInput.svelte
- [x] SearchLink.svelte

### PR DESCRIPTION
**Pull request by eyeseast**
Basic search bar stories, now with mocked data.

Closes #369

### CODE REVIEW

**Review by allanlasser — APPROVED** [review]

File: src/pages/app/stories/SearchBar.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  parameters={{
+    msw: {
+      handlers: [mock.users.data, mock.organizations.data, mock.projects.data],
+    },
+  }}
+/>
+
+<Story
+  name="dialog"
+  args={{ dialog: true, search: "project:example-project-123" }}
```
Comment (allanlasser) [inline src/pages/app/stories/SearchBar.stories.svelte:51]: This story is missing the data handlers

File: src/pages/app/stories/SearchBar.stories.svelte [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <SearchBar {...args} />
+  </div>
+</Template>
+
+<Story
+  name="default"
+  args={{ search: "project:example-project-123" }}
+  parameters={{
+    msw: {
+      handlers: [mock.users.data, mock.organizations.data, mock.projects.data],
```
Comment (allanlasser) [inline src/pages/app/stories/SearchBar.stories.svelte:24]: User and organization mocking is working as expected, but typing `project:` doesn't trigger any network requests.

**Review by allanlasser — COMMENTED** [review]
Data mocking is great, caught two small things

File: src/pages/app/stories/SearchBar.stories.svelte (hunk 2 above)
Comment (eyeseast) [reply src/pages/app/stories/SearchBar.stories.svelte:24]: This seems to also be an issue in production, so let's consider it a bug to be fixed.

**Pull request merged**